pkg/expr/mathexp/parse/node.go

```diff
@@ -405,6 +405,8 @@ const (
 	TypeVariantSet
 	// TypeNoData is a no data response without a known data type.
 	TypeNoData
+	// TypeRowData is a data response from SQL Expression.
```

Comment:
@kylebrandt -- is this essentially an "i don't know" table format?

Reply:
Seems to be, as it doesn't support binary or operations. I think we could just call it table

Reply:
updated